app/javascript/components/public_health/AdvancedFilter.js

```diff
@@ -202,7 +204,7 @@ class AdvancedFilter extends React.Component {
       this.setState({ savedFilters: response.data }, () => {
         // Apply filter if it exists in local storage
         let sessionFilter = localStorage.getItem(`SaraFilter`);
-        if (parseInt(sessionFilter)) {
+        if (this.props.useLocalStorage && parseInt(sessionFilter)) {
```

Comment:
I purposely didn't add that check there in case a user deletes the current saved filter. I guess that could also be handled by checking to see if the filter exists when trying to load filters and deleting it if it doesn't exist

Reply:
https://github.com/SaraAlert/SaraAlert/pull/562/commits/1f13143f8968feff3c7e9ada6139b47e5cd4ecff#diff-b16a4080191131abd62e13c342100314e27e185f75ec98667d6635351bfb3ea8R408

Reply:
tested deleted saved filter from custom export and reloading page and it wasn't a problem